src/popup.coffee

```diff
@@ -1,6 +1,8 @@
 class SomaPlayerPopup
+
   constructor: ->
-    console.debug 'popup opened'
+    @base = this
+    @station_list = @fetch_soma_channels
```

Comment:
This isn't actually calling the method, but I don't see this variable getting used so I think you can just delete the line.